gitea/release.go

```diff
@@ -46,6 +47,33 @@ func (c *Client) GetRelease(user, repo string, id int64) (*Release, error) {
 	return r, err
 }
 
+// ListReleaseAssets gets all the assets of a release in a repository
+func (c *Client) ListReleaseAssets(user, repo string, id int64) (*Release, error) {
```

Comment:
@bkcsoft I think it will be confusing that endpoints and in json it is called assets but in sdk attachments